src/alert/base.rs

```diff
@@ -217,17 +223,19 @@ pub trait AlertWorker {
         object_id: impl Into<Self::ObjectId> + Send,
         ra: f64,
         dec: f64,
-        prv_candidates_doc: &Vec<mongodb::bson::Document>,
-        prv_nondetections_doc: &Vec<mongodb::bson::Document>,
-        fp_hist_doc: &Vec<mongodb::bson::Document>,
+        prv_candidates_doc: &Vec<Document>,
+        prv_nondetections_doc: &Vec<Document>,
+        fp_hist_doc: &Vec<Document>,
+        survey_matches: &Document,
```

Comment:
might just want to make it an option if it's not relevant for all surveys, maybe better than having to create an empty document as I saw in `lsst.rs`?